_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
+- Metadata
+- Configuration
+- Security
+- Networking
+- Storage
+- Cluster Operation
+- Workloads
+- Architecture
+- API
+- Logging
+- Extensions
+
+# TODO: Add list of glossary terms here
+# (to avoid extra loop logic on pages that need the mapping of words to tags)
+# e.g.
+# terms:
+# - name: StatefulSets
+#   tags:
```

Comment:
@chenopis I think that's possible---after doing the term MVP we can see what this might look like for pages